torch/csrc/jit/passes/quantization/quantization_type.h

```diff
@@ -5,7 +5,7 @@ namespace jit {
 
 // Quantization type (dynamic quantization, static quantization).
 // Should match the Python enum in quantize_script.py
-enum class QuantType{ DYNAMIC, STATIC };
+enum class QuantType{ DYNAMIC, STATIC, QAT };
```

Comment:
I was mostly trying to say that QAT might just be sharing all the passes with Static, but it's probably fine since we also share most of the static code in dynamic as well.

Reply:
yeah, hopefully most of the code can be shared.  It's mostly for readability, so that we state intent explicitly that something works for both static and qat, such as

```
if (quant_type == QuantType::STATIC || quant_type == QuantType::QAT) {
  ...
}
```

Reply:
that might be around the whole code base, I think we should just do the check whenever we have special path for QAT